hack/install.sh

```diff
@@ -84,9 +84,21 @@ install_cni_plugins() {
 }
 
 install_containerd() {
-  CONTAINERD_VER=v1.6.4
+
   $SUDO systemctl unmask containerd || :
-  $SUDO $ARKADE system install containerd --systemd --version ${CONTAINERD_VER}  --progress=false
+
+  CONTAINERD_VER=1.6.4
+  arch=$(uname -m)
+  if[ $arch == "armv7l"]; then
```

Comment:
Currently the script fails:
```
line 92: syntax error near unexpected token `then'
```

```suggestion
  if [ $arch == "armv7l"]; then
```

Reply:
Concur.  Saw the same.